### PR TITLE
Added demo gif animation of the sample

### PR DESCRIPTION
**Pull request by AndroidDeveloperLB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This folder contains the source code for a [Google I/O 2016 codelab on Android N Quick Settings](https://codelabs.developers.google.com/codelabs/android-n-quick-settings/#0).
 
+## Demo
+
+![enter image description here](https://github.com/AndroidDeveloperLB/android-n-quick-settings/blob/master/extras/demo.gif?raw=true)
```
Comment (ngzhian) [inline README.md]: Can this be changed to "Demo of Quick Settings tile"?

**Review by ngzhian — CHANGES_REQUESTED** [review]

**Comment by AndroidDeveloperLB** [comment]
It's the demo of this repo, no? You want that if the demo will include more options in the future, this gif will be only about this part of the demo?

**Comment by ngzhian** [comment]
Yes good point, let's say "Demo of Quick Settings tile in tutorial".
To clarify, I meant to change the image description, right now it sounds like a placeholder.

**Comment by AndroidDeveloperLB** [comment]
Oh you mean the inner description of the image? The one that appears only when hovering the mouse of it?
If so, ok.
I thought you meant the title...

**Comment by ngzhian** [comment]
Yes I meant the inner description. Sorry I could have been more clear :)

**Comment by AndroidDeveloperLB** [comment]
So sure, add there what you wish.
Here, I've updated it:
https://github.com/googlecodelabs/android-n-quick-settings/compare/master...AndroidDeveloperLB:master

**Comment by ngzhian** [comment]
Perfect! Thanks :)

**Comment by AndroidDeveloperLB** [comment]
Good I removed the repo

**Pull request merged**